vaadin-testbench-unit/src/main/java/com/vaadin/testbench/unit/Scan.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+package com.vaadin.testbench.unit;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ ElementType.TYPE, ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Scan {
```

Comment:
The name could be anything or nothing.. could this be more elaborative like `@TestBenchUIScan` if it meant to be used by developer to specify their custom wrapper packages.

Reply:
TestBenchUIScan sounds like it scans for UI implementations.
Renamed to `@ComponentWrapPackages`